lighthouse-core/scripts/pptr-run-devtools.js

```diff
@@ -117,7 +117,17 @@ const startLighthouse = `
 
   // Give the main target model a moment to be available.
   // Otherwise, 'SDK.TargetManager.TargetManager.instance().mainTarget()' is null.
-  await new Promise(resolve => setTimeout(resolve, 1000));
+  const targetManager =
+    SDK.targetManager || (SDK.TargetManager.TargetManager || SDK.TargetManager).instance();
+  if (targetManager.mainTarget() === null) {
+    if (targetManager.observeTargets) {
```

Comment:
When would this be falsy? In an older version of DT?

Reply:
Could be. It was pretty involved making sure a bunch of older chromes worked with this script, so I am doing without testing if it is necessary 😬